deck-of-cards-kata-solutions/src/test/java/bnymellon/codekatas/deckofcards/list/immutable/ApacheCommonsDeckOfCardsAsListTest.java

```diff
@@ -109,8 +111,12 @@ public void shuffleAndDealHands()
         var jdkHands = this.jdkDeck.shuffleAndDeal(new Random(1), 5, 5);
         var acHands = this.acDeck.shuffleAndDeal(new Random(1), 5, 5);
         Assertions.assertEquals(jdkHands, acHands);
-        var hands = acHands.stream().map(each ->
-                IterableUtils.toString(each.stream().sorted().toList(), Object::toString, ", ", "", "")).collect(Collectors.toList());
+        var hands = FluentIterable.of(acHands)
```

Comment:
Is the purpose of switching to FluentIterable to not rely on stream()?

Reply:
Yes. These tests are for Apache Commons, as much as possible I want to use native features in the library.

Reply:
Makes sense. LGTM and approved!